packages/driver/src/cypress/error_messages.ts

```diff
@@ -1694,7 +1694,7 @@ export default {
       message: `${cmd('switchToDomain')} requires enabling the experimentalMultiDomain flag`,
     },
     invalid_domain_argument: {
-      message: `${cmd('switchToDomain')} requires the first argument to be a string. You passed: \`{{arg}}\``,
+      message: `${cmd('switchToDomain')} requires the first argument to be a string representing a domain that does not not include any subdomains. You passed: \`{{arg}}\``,
```

Comment:
Are we able to tell them what we expected the domain to be in the error message?

Reply:
I've updated the error message to be more descriptive generally, but haven't tailored the message to tell them how to correct whatever string they've passed, I think that might be going too far.